ttnn/cpp/ttnn/operations/moreh/moreh_clip_grad_norm/moreh_clip_grad_norm_step2/device/moreh_clip_grad_norm_step2_device_operation.cpp

```diff
@@ -37,7 +37,7 @@ void MorehClipGradNormStep2Operation::validate_on_program_cache_hit(
 
 MorehClipGradNormStep2Operation::shape_return_value_t MorehClipGradNormStep2Operation::compute_output_shapes(
     const operation_attributes_t& operation_attributes, const tensor_args_t& tensor_args) {
-    return SimpleShape{tt::constants::TILE_HEIGHT, tt::constants::TILE_WIDTH};
+    return SimpleShape{1, 1};
```

Comment:
Kind of magic numbers, please comment out their meaning if possible

Reply:
resolved in [ce8237a](https://github.com/tenstorrent/tt-metal/pull/15897/commits/ce8237a7c45fd186600377fcd4f57975e5e5fd4e)